src/main/java/com/simibubi/create/content/trains/GlobalRailwayManager.java

```diff
@@ -220,8 +219,9 @@ private void tickTrains(Level level) {
 		// keeping two lists ensures a tick order starting at longest waiting
 		for (Train train : waitingTrains)
 			train.earlyTick(level);
-		for (Train train : movingTrains)
-			train.earlyTick(level);
+		for (Train train : movingTrains){
+			train.updateCollisionCache();
+			train.earlyTick(level);}
```

Comment:
formatting please
```java
for (Train train : movingTrains) {
	train.updateCollisionCache();
	train.earlyTick(level);
}
```

do we not need the collision cache for waiting trains? do we not expect moving trains to collide with trains that are stationary?